test/functional/api/cas/version.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright(c) 2019-2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import os
+
+from api.cas import git
+
+
+class CasVersion:
```

Comment:
maybe you can extend existing python  Version class? This class has all comparison methods implemented etc